components/list.html

```diff
@@ -0,0 +1,89 @@
+        </button>
+    </nav>
+    <div class="main-grid">
+        <div class="side-nav padding-10 margin-3">
+            <a href="/documentation.html">
+                <li class="padding-7">Installation</li>
+            </a>
+            <a href="avatar.html">
+                <li class="padding-7">Avatar</li>
+            </a>
+            <a href="alert.html">
+                <li class="padding-7">Alert</li>
+            </a>
+            <a href="badge.html">
+                <li class="padding-7">Badge</li>
+            </a>
+            <a href="card.html">
+                <li class="padding-7">Card</li>
+            </a>
+            <a href="image.html">
+                <li class="padding-7">Image</li>
+            </a> <a href="input.html">
+                <li class="padding-7">Input</li>
+            </a>
+            <a href="list.html">
+                <li class="padding-7 active">List</li>
+            </a>
+            <a href="modal.html">
+                <li class="padding-7">Modal</li>
+            </a>
+            <a href="navigation.html">
+                <li class="padding-7">Navigation</li>
+            </a>
+            <a href="grid.html">
+                <li class="padding-7">Grid</li>
+            </a>
+        </div>
+        <div class="components margin-adjust width-3/5">
+            <h2 class="margin-13">Stacked List</h2>
+            <div class="stacked-list margin-9">
+                <div class="noti-user-list padding-1"><img
+                        src="https://images.unsplash.com/photo-1506464253090-7af30fe60a71?ixid=MXwxMjA3fDB8MHxzZWFyY2h8Mnx8Y2lyY2xlJTIwaW1hZ2UlMjBodW1hbnxlbnwwfHwwfA%3D%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=900&q=60"
+                        class="list-user"><span class="user-action">Robert called you</span></div>
+                <div class="noti-user-list padding-1 alt"><img
+                        src="https://images.unsplash.com/photo-1615125946484-86dd0a2cdb18?ixid=MXwxMjA3fDB8MHxzZWFyY2h8MjB8fGNpcmNsZSUyMGltYWdlJTIwaHVtYW58ZW58MHx8MHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=900&q=60"
+                        class="list-user"><span class="user-action">Rachel sent a text</span></div>
+                <div class="noti-user-list padding-1"><img
+                        src="https://images.unsplash.com/photo-1599140781162-68659a79e313?ixid=MXwxMjA3fDB8MHxzZWFyY2h8MjF8fGNpcmNsZSUyMGltYWdlJTIwaHVtYW58ZW58MHx8MHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=900&q=60"
+                        class="list-user"><span class="user-action">Sarah left a voice recording</span></div>
+                <div class="noti-user-list padding-1 border-bottom alt"><img
+                        src="https://images.unsplash.com/photo-1559843811-a733d94b6f8a?ixid=MXwxMjA3fDB8MHxzZWFyY2h8MjR8fGNpcmNsZSUyMGltYWdlJTIwaHVtYW58ZW58MHx8MHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=900&q=60"
+                        class="list-user"><span class="user-action">Monica missed your call</span></div>
```

Comment:
use prettier or beautify which will help you to auto-format your code on save both are available for VS code